setup.py

```diff
@@ -11,11 +11,7 @@
 
 import os
 import sys
-try:
-    from setuptools import setup, Extension
-except ImportError:
-    from distutils.core import setup, Extension
-
+from setuptools import setup, Extension
```

Comment:
Why this? AFAIK setuptools is a third-party dep which is not installed by default.